lib/matplotlib/type1font.py

```diff
@@ -309,7 +309,7 @@ def suppress(tokens):
 
         while True:
             token, value = next(tokens)
-            if token == 'name' and value in table:
+            if token is cls._name and value in table:
```

Comment:
Ah, sorry :sheep: makes sense now.

Was this the stuff that was a long time broken do to `token` no longer being a string?